roxctl/common/flags/hide.go

```diff
@@ -0,0 +1,23 @@
+package flags
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	"github.com/stackrox/rox/pkg/set"
+)
+
+// HideInheritedFlags hides all inherited flags except the flag names given.
+// This is especially useful for commands that do not require central connectivity and want to avoid exposing these
+// flags to the client to avoid confusion.
+func HideInheritedFlags(cmd *cobra.Command, flagsToShow ...string) {
```

Comment:
❤️ 

How about extracting this to a separated PR and add to 
- `version`
- `help`
- `completion`

Reply:
Help won't sadly work since its a built-in command by cobra :/

Nevertheless, created the follow-up PR: #5476